ui/components/MesheryMeshInterface/PatternService/RJSF.js

```diff
@@ -18,26 +33,51 @@ function RJSFButton({ handler, text }) {
   );
 }
 
-function RJSF({ jsonSchema, onChange, hideSubmit, hideTitle, onSubmit, onDelete }) {
+const uiSchema = {
+  replicas: {
+    "ui:widget" : "range"
+  }
+};
+
+function RJSF({ jsonSchema, onChange, hideSubmit, hideTitle, onSubmit, onDelete, isMeshery }) {
   const [data, setData] = React.useState();
 
   React.useEffect(() => {
     onChange?.(data);
   }, [data]);
 
   return (
-    <Form
-      schema={hideTitle ? deleteTitleFromJSONSchema(jsonSchema) : jsonSchema}
-      idPrefix={jsonSchema?.title}
-      onChange={(e) => setData(e.formData)}
-      formData={data}
-      liveValidate
-      additionalMetaSchemas={[JS4]}
-      // noHtml5Validate
-    >
-      {hideSubmit ? true : <RJSFButton handler={onSubmit} text="Submit" />}
-      {hideSubmit ? true : <RJSFButton handler={onDelete} text="Delete" />}
-    </Form>
+    <>
```

Comment:
is this necessary? @Jashpatel1

Reply:
Coz single div inside return statement.

Reply:
the rule is single root component if I am not wrong and in either of the cases if `isMeshery` is true or false there is only one root component right?

Reply:
Yes, but the condition is itself JSX expression and should be enclosed inside a root element

Reply:
ah my bad cool